### PR TITLE
DCS-1332 Adding transfers endpoint

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/Transfer.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  @Schema(description = "PNC number", example = "01/1234X")
+  val pncNumber: String?,
+
+  @Schema(description = "Date expected", example = "2020-02-23")
+  val date: LocalDate,
+
+  @Schema(description = "From Location", example = "Kingston-upon-Hull Crown Court")
+  val fromLocation: String,
+)
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/Transfer.kt:31]: like an arrival without irrelevant info - no `id`, `fromLocationType` etc..

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/TransfersServiceTest.kt [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+    assertThatThrownBy {
+      transfersService.getTransfer("NMI", "A1234AA")
+    }.isEqualTo(NotFoundException("Could not find transfer with prisonNumber: 'A1234AA'"))
+  }
+
+  companion object {
+
+    private val arrivalKnownToNomis = OffenderMovement(
+      offenderNo = "G6081VQ",
```
Comment (louisenorris) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/TransfersServiceTest.kt:73]: Should offenderNo here also be changed to prisonNumber for consistency?

**Review by louisenorris — APPROVED** [review]
LGTM

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/TransfersServiceTest.kt (hunk 2 above)
Comment (andrewrlee) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/TransfersServiceTest.kt:73]: yes, as a follow up

**Pull request merged**